chemprop/cli/train.py

```diff
@@ -638,6 +641,63 @@ def build_splits(args, format_kwargs, featurization_kwargs):
     return train_data, val_data, test_data
 
 
+def summarize(args, dataset: _MolGraphDatasetMixin) -> tuple[list, list]:
+    if args.task_type == "spectral":
+        pass
+    elif args.task_type in ["regression", "regression-mve", "regression-evidential"]:
+        y = dataset.Y
+        y_mean = y.mean()
+        y_std = y.std()
+        y_median = np.median(y)
+        mean_dev_abs = np.abs(y - y_mean)
+        frac_1_sigma = (mean_dev_abs < y_std).sum() / len(y)
+        frac_2_sigma = (mean_dev_abs < 2 * y_std).sum() / len(y)
+        column_headers = ["Statistic", "Value"]
+        table_rows = [
+            ["Num. datapoints", f"{len(y)}"],
+            ["Mean", f"{y_mean:0.2f}"],
+            ["Std. dev.", f"{y_std:0.2f}"],
+            ["Median", f"{y_median:0.2f}"],
+            ["% within 1 s.d.", f"{frac_1_sigma:0.0%}"],
+            ["% within 2 s.d.", f"{frac_2_sigma:0.0%}"],
+        ]
+        return (column_headers, table_rows)
+    elif args.task_type in [
+        "classification",
+        "classification-dirichlet",
+        "multiclass",
+        "multiclass-dirichlet",
+    ]:
+        y = dataset.Y
+        mask = np.isnan(y)
+        classes = np.sort(np.unique(y[~mask]))
+        class_counts = (classes[:, None] == y[:, 0]).sum(1)
+        class_fracs = class_counts / len(y)
+        nan_count = (mask[:, 0]).sum()
+        nan_frac = nan_count / len(y)
+        column_headers = ["Class", "Count", "Percent"]
+        table_rows = [
+            (f"{k}", f"{class_counts[i]}", f"{class_fracs[i]:0.0%}") for i, k in enumerate(classes)
+        ]
+        table_rows.append(["NAN", f"{nan_count}", f"{nan_frac:0.0%}"])
+        table_rows.append(["Total", f"{len(y)}", f"{100.00}%"])
+        return (column_headers, table_rows)
+    else:
+        raise ValueError(f"unsupported task type! Task type '{args.task_type}' was not recognized.")
```

Comment:
```suggestion
        raise ValueError(f"Unsupported task type! Task type '{args.task_type}' was not recognized.")
```

Reply:
```python
>>> raise ValueError("unsupported task type!")
ValueError: unsupported task type!
>>> raise ValueError("Unsupported task type!")
ValueError: Unsupported task type!
```

Most of the code base opts for the latter, but I'd argue the former (not capitalizing) is more correct as it's more consistent with Python style:
```python
>>> float('s')
ValueError: could not convert string to float: 's'
```

Reply:
I'll defer to you on this to follow what the standard library does. My suggestion here was motivated after I did a quick search of "valueerror" in the Numpy Github and saw this [example](https://github.com/numpy/numpy/blob/2f3da0d2dd14041e0a4a39b996380511039bfc51/numpy/random/_mt19937.pyx#L182) where their error message is capitalized. 
`raise ValueError("Seed must be between 0 and 2**32 - 1")`

Reply:
NumPy is inconsistent, see [here](https://github.com/numpy/numpy/blob/main/numpy/_core/arrayprint.py#L82):
```python
        raise ValueError("floatmode option must be one of " +
```
At least main python style is consistent AFAICT